README.md

```diff
@@ -1 +1,186 @@
+* [Installing the plugin](#installing-the-plugin)
+   * [Easiest Install](#easiest-install)
+   * [Temporary Install](#temporary-install)
+   * [(more) Permanent Install](#more-permanent-install)
+   * [(most) Permanent Install](#most-permanent-install)
+* [The Fine Print](#the-fine-print)
+* [Getting Help](#getting-help)
+
+## Developing
+
+### Requirements
+
+* [Apache Maven 3.3.3+](https://maven.apache.org/install.html)
+* [Java 8+](http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html)
+* Network access to https://repository.sonatype.org/content/groups/sonatype-public-grid
+
+Also, there is a good amount of information available at [Bundle Development](https://help.sonatype.com/display/NXRM3/Bundle+Development)
+
+### Building
+
+To build the project and generate the bundle use Maven
+
+    mvn clean package -PbuildKar
+
+If everything checks out, the bundle for apk should be available in the `target` folder
+
+#### Build with Docker
+
+`docker build -t nexus-repository-apk:0.0.1 .`
+
+#### Run as a Docker container
+
+`docker run -d -p 8081:8081 --name nexus nexus-repository-apk:0.0.1` 
+
+For further information like how to persist volumes check out [the GitHub repo for our official image](https://github.com/sonatype/docker-nexus3).
+
+The application will now be available from your browser at http://localhost:8081
+
+## Using APK With Nexus Repository Manager 3
+
+[We have detailed instructions on how to get started here!](docs/APK_USER_DOCUMENTATION.md)
+
+## Compatibility with Nexus Repository Manager 3 Versions
+
+The table below outlines what version of Nexus Repository the plugin was built against
+
+| Plugin Version | Nexus Repository Version |
+|----------------|--------------------------|
+| v0.0.1         | 3.19.0-SNAPSHOT          |
+
+If a new version of Nexus Repository is released and the plugin needs changes, a new release will be made, and this
+table will be updated to indicate which version of Nexus Repository it will function against. This is done on a time 
+available basis, as this is community supported. If you see a new version of Nexus Repository, go ahead and update the
+plugin and send us a PR after testing it out!
+
+All released versions can be found [here](https://github.com/sonatype-nexus-community/nexus-repository-apk/releases).
+
+## Features Implemented In This Plugin 
+
+| Feature | Implemented          |
+|---------|----------------------|
+| Proxy   | :heavy_check_mark: * |
+| Hosted  |                      |
+| Group   |                      |
+
+## Installing the plugin
+
+There are a range of options for installing the apk plugin. You'll need to build it first, and
+then install the plugin with the options shown below:
+
+### Easiest Install
+
+Thanks to some upstream work in Nexus Repository (versions newer than 3.15), it's become a LOT easier to install a plugin. To install the `apk` plugin, follow these steps:
```

Comment:
NOTE: Since this came out after 3.15, I wonder if we need
1) the parenthetical reference to 3.15
2) the permanent methods.
If you strike the permanent methods, could just rename this permanent install (rather than easiest install).
Just a ponderance, thus NOTE.

Reply:
I think we might be able to get rid of a few of the install methods, or link to a common place with like "You can do this too, but you don't need to anymore"

Reply:
@joedragons @DarthHater Sounds good! I'll get rid of `More` and `Most Permanent Install`s

Reply:
in: https://github.com/sonatype-nexus-community/nexus-repository-apk/pull/1/commits/09471e269484ac8a27a67f73695f1a82de8bc47c